include/cuco/detail/bitwise_compare.cuh

```diff
@@ -16,6 +16,8 @@
 
 #pragma once
 
+#include <cuco/traits.hpp>
```

Comment:
I just created #248 to fix this bug since the whole chain of refactoring PRs would take a while to merge into the default branch. Thanks for catching this issue!